admin/js/wizard.js

```diff
@@ -119,7 +121,54 @@ function setOption(option) {
       showMessage(`Error setting option: ${option.key}`, 'error');
     });
 }
-
+function onClickOptionButton(event) {
+  if (event){
+    event.preventDefault();
+  }
+  var formData = new FormData(event.target, event.submitter);
+  const button = formData.get('button');
+  if (button !== 'all') {
+  	try {
+      setOption(
+        {
+          "key":button,
+          "value":formData.get(button)
+        }
+      );
+    } catch (error) {
+      console.error(error);
+      showMessage('Error setting option', 'error');
+    }
+  } else{
+    for (entry of formData.entries()) {
+      try {
+        if (entry.value !== 'all') {
+          setOption(
+            {
+              "key":entry[0],
+              "value":entry[1]
+            }
+          );
+        }
+      } catch (error) {
+        console.error(error);
+        showMessage('Error setting option', 'error');
+      }
+    }
+  }
+}
+function loopSetOptions(event) {
+  var button = event.target;
+  var options = button.dataset.options;
+  for (option of options) {
+    try {
+      setOption(option);
+    } catch (error) {
+      console.error(error);
+      showMessage('Error setting option', 'error');
+    }
+  }
+}
```

Comment:
What are these loop functions for? I don't see them used anywhere, so I think they can be removed.